tests/pytests/test_f12_ints.py

```diff
@@ -0,0 +1,50 @@
+"""
+This file tests the F12 two-electron integrals from libmints
+"""
+import numpy as np
+import pytest
+import psi4
+from pathlib import Path
+import json
+
+
+pytestmark = [pytest.mark.api, pytest.mark.quick]
+
+
+@pytest.fixture
+def reference_data():
+    # Reference data generated using Psi4 1.5, still using OS86
+    with open(Path(__file__).parent / "f12_libint1.json") as f:
+        reference_data = json.load(f)
+    return reference_data
+    
+def matlist_to_ndarray(mats):
+    """Converts a list of psi4.core.Matrix to np.ndarray"""
+    return psi4.driver.p4util.numpy_helper._to_array(mats) 
+
+def test_f12_integrals(reference_data):
+    assert reference_data['version'] == '1.5'
+    refdata = reference_data['data']
+    moldata = refdata['h2o']
+    mol = psi4.core.Molecule.from_string(moldata.pop('psi4string'))
+
+    basis = psi4.core.BasisSet.build(mol, 'orbital', '6-31G*')
+    mints = psi4.core.MintsHelper(basis)
+    f12 = mints.f12_cgtg(1.0)
+
+    for int_type in ['F', 'F2', 'FG', 'Uf']:
+        ref = moldata[int_type]
+        shape = ref[f'L_shape']
```

Comment:
`L_shape` does not make too much sense, maybe just `shape`? I chose the name because of ang momentum... But it doesn't matter that much 😄

Reply:
Done. Changed to shape and changed the integral label from `L` to `I`.